quartz/components/ContentMeta.tsx

```diff
@@ -30,7 +30,7 @@ export default ((opts?: Partial<ContentMetaOptions>) => {
       const segments: (string | JSX.Element)[] = []
 
       if (fileData.dates) {
-        segments.push(formatDate(getDate(cfg, fileData)!, cfg.locale))
+        segments.push(<Date date={getDate(cfg, fileData)!} locale={cfg.locale} />)
```

Comment:
this seems like a separate change. I saw a separate PR for this.

Let's remove this from it.

Reply:
removed